patterns/httpd

```diff
@@ -0,0 +1,15 @@
+HTTPDUSER %{EMAILADDRESS}|%{USER}
+HTTPDERROR_DATE %{DAY} %{MONTH} %{MONTHDAY} %{TIME} %{YEAR}
+
+# Log formats
+HTTPD_COMMONLOG %{IPORHOST:clientip} %{HTTPDUSER:ident} %{USER:auth} \[%{HTTPDATE:timestamp}\] "(?:%{WORD:verb} %{NOTSPACE:request}(?: HTTP/%{NUMBER:httpversion})?|%{DATA:rawrequest})" %{NUMBER:response} (?:%{NUMBER:bytes}|-)
```

Comment:
If we do change, for example, to use `%{NUMBER:bytes:int}` we'll want the specs to expect numbers, not strings.

For background, I've wanted bytes (especially) and httpversion stored as numbers in Elasticsearch for years, but I didn't want to break compat. This PR is a perfect opportunity to do that.

Thoughts?